src/components/CommentItem.jsx

```diff
@@ -1,10 +1,26 @@
-const CommentItem = ({comment}) => {
+import { useContext } from "react";
+import { UserContext } from "./contexts/UserContext";
+import { deleteComment } from "../api";
+
+const CommentItem = ({article, comment, setComments}) => {
+    const {user} = useContext(UserContext)
+    
+    const handleDelete = (id) => {
+        deleteComment(id)
```

Comment:
we need a .catch to handle the event that the comment fails to delete - we want to undo our optimistic rendering here